packages/mdc-menu/README.md

```diff
@@ -239,7 +239,7 @@ mdc.MDCSimpleMenu.attachTo(document.querySelector('.mdc-simple-menu'));
 Simple menus can easily be initialized using their default constructors as well, similar to `attachTo`.
 
 ```javascript
-import MDCSimpleMenu from 'mdc-menu';
+import MDCSimpleMenu from '@material/mdc-menu';
```

Comment:
Is this supposed to be `import {MDCSimpleMenu}`? I don't think our component modules have a default export.